src/test/java/org/opentripplanner/updater/vehicle_rental/datasources/GbfsFeedLoaderTest.java

```diff
@@ -73,6 +84,33 @@ void getV10FeedWithExplicitLanguage() {
         validateV10Feed(loader);
     }
 
+    @Test
+    @Ignore
+    void fetchAllPublicFeeds() throws IOException {
+        InputStream is = HttpUtils.getData("https://raw.githubusercontent.com/NABSA/gbfs/master/systems.csv");
+        CsvReader reader = new CsvReader(is, StandardCharsets.UTF_8);
+        reader.readHeaders();
+        List<Exception> exceptions = new ArrayList<>();
+
+        while (reader.readRecord()) {
+            try {
+                String url = reader.get("Auto-Discovery URL");
+                if (url.contains("gbfs.spin.pm"))
+                new GbfsFeedLoader(url, Map.of(), null).update();
+            } catch (Exception e) {
+                exceptions.add(e);
+            }
+
+        }
+        assertTrue(exceptions.isEmpty(), exceptions.stream().map(Exception::getMessage).collect(Collectors.joining("\n")));
+    }
+
+    @Test
+    @Ignore
+    void testSpin() {
+        new GbfsFeedLoader("https://gbfs.spin.pm/api/gbfs/v2_2/edmonton/gbfs", Map.of(), null).update();
```

Comment:
Do we want to add tests that test against external endpoints as they have tendency to be unavailable once in a while?

Reply:
This test is not run by default, as it has an `@Ignore` annotation

Reply:
Ah sorry missed that but there is another test currently failing that seems to be similar

Reply:
Ah, seems that I used the annotation from Junit 4, while tests are using Junit 5. This should be fixed now